src/evaluator/functions.ts

```diff
@@ -456,11 +456,47 @@ diff.changedOnly = () => {
 }
 diff.changedOnly.arity = 3
 
+const array: FunctionSet = {}
+
+array.unique = async function (args, scope, execute) {
+  if (args.length > 1) {
+    throw new Error('Incorrect number of arguments to function "array::unique"')
+  }
```

Comment:
This should be handled by setting `array.unique.arity = 1` instead. That will correctly treat `array::unique(foo, bar)` as a compile-time error.